docs/source/advanced/filters.md

```diff
@@ -24,6 +24,10 @@ You can combine search options on text fields by writing `;`. For example `Alex;
 
 Filters and ordered columns set for an entity list are saved into your session, so as long as you stay connected you don't need to re-set them on every page.
 
+## Location name
+
+You can filter entities based on their location. The search field looks for entities that have a location with that exact name. You can use the selection box at the right of the field to select between 4 different behaviours for the filter, these are: include, which includes all entities in that location; exclude, which excludes all entities in that location; with children, which includes all entities in that location or any of all its children locations, and none, which shows all entities without a defined location.
```

Comment:
It would be easier to read if all 4 options are each on a single bullet point
```
* **Include**: includes all...
* **Exclude**: excludes all...
```